cmd/postgres_exporter/postgres_exporter.go

```diff
@@ -702,9 +702,15 @@ func (e *Exporter) checkMapVersions(ch chan<- prometheus.Metric, server *Server)
 	versionDesc := prometheus.NewDesc(fmt.Sprintf("%s_%s", namespace, staticLabelName),
 		"Version string as reported by postgres", []string{"version", "short_version"}, server.labels)
 
+	var short_version = semanticVersion.String()
```

Comment:
This needs to follow Go naming conventions.

```suggestion
	var semanticVersionShort = semanticVersion.String()
```

Reply:
Changed this now in ce2f9089bd